x-pack/test/functional_solution_sidenav/tests/search_sidenav.ts

```diff
@@ -48,17 +48,231 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
           deepLinkId: 'enterpriseSearch',
         });
 
-        // check the Content > Indices section
+        // Verify all expected top-level links exist
```

Comment:
Could you move these new tests to a place/folder owned by the "search" solution? This file is for SharedUX smoke tests to make sure each solution side nav renders correctly.

Reply:
will do

Reply:
@sebelga done, moved to the enterprise search tests.